lib/fluent/plugin/out_mail.rb

```diff
@@ -184,8 +184,8 @@ def sendmail(subject, msg)
     subject = subject.force_encoding('binary')
     body = msg.force_encoding('binary')
 
-    if time_locale
-      date = Time::now.timezone(time_locale)
+    if @time_locale
```

Comment:
`config_param` will define an accessor to the instance variable, so you should have an access to `time_locale`. 
Can you give me a test case or an example of conf to reproduce your case?